lib/typescript/render/providers/facebook/facebookModel.ts

```diff
@@ -69,4 +88,11 @@ export interface GenericTemplate extends Content {
 }
 
 // Add a new facebook content model here:
-export type ContentUnion = TextContent | ImageContent | ButtonTemplate | GenericTemplate;
+export type ContentUnion =
+  | TextContent
+  | ImageContent
+  | VideoContent
+  | ButtonTemplate
+  | GenericTemplate
+  | QuickRepliesContent;
+export type AttachmentUnion = TextContent | ImageContent | VideoContent | ButtonTemplate | GenericTemplate;
```

Comment:
I created AttachmentUnion because Facebook attachment can be image, audio, video, file or template (but not QuickReplies).